src/Components/SuccessPopUp/index.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react';
+import styles from './style.module.css';
+
+function SuccessPopUp() {
+  const [show, setShow] = useState(true);
+  const disapear = () => setShow(false);
+  // TODO: Create route for "Undo" button
+  // Error message is the only prop
+
+  return (
+    <div>
+      {show && (
+        <div className={styles.container}>
+          <div className={styles.topBar}>
+            APPLICATION STATUS CHANGE
+          </div>
+          <div className={styles.mainBody}>
+            <p>
+              The application status has been successfully updated.
```

Comment:
please pass this in as a variable like you did in the ErrorPopUp. 
e.g.  `props.successMessage` 
and give this hardcoded message as the default val like you did above.